docs/user-guide/01-blueprint-reference.md

```diff
@@ -107,6 +93,31 @@ version = "*"
 name = "openssh-server"
 version = "*"
 ```
+</TabItem>
+<TabItem value="hosted">
+```json
+{
+  "image_name": "tmux",
+  "image_description": "tmux image with openssh",
+  "distribution": "fedora-38",
+  "customizations": {
+    "packages": ["tmux", "openssh-server"]
+  }
+}
+```
+
```

Comment:
Maybe add a line of explanation here like
"Example for Red Hat Enterprise Linux"
so have some separation, but that's no blocker for me

Reply:
@schuellerf You mean because the two distros are different? Why are they different in the first place? :thinking:

Reply:
No just to visually separate those examples a little more, and the two examples make sense because the syntax of the versioning is different.
Alternatively we could somehow reference the list of possible distros&versions

Reply:
So, I initially added a RHEL example when modifying the docs for dot-notation, because one can hardly showcase the dot notation on Fedora - it does not use minor release version :slightly_smiling_face: 

Happy to add the line, but the intention here is not to showcase RHEL as such, but the dot-notation. We could also just remove the Fedora example and have only RHEL :shrug:

Reply:
Let's not overthink it - happy to add more examples later. This makes sense.